src/renderer/components/drawer/drawer-item.tsx

```diff
@@ -5,14 +5,13 @@
 
 import "./drawer-item.scss";
 import React from "react";
-import { cssNames, displayBooleans } from "../../utils";
+import { cssNames } from "../../utils";
 
 export interface DrawerItemProps extends React.HTMLAttributes<HTMLDivElement> {
   name: React.ReactNode;
   title?: string;
   labelsOnly?: boolean;
   hidden?: boolean;
-  renderBoolean?: boolean; // show "true" or "false" for all of the children elements are "typeof boolean"
```

Comment:
Technically this is a breaking change (when building an extension) but I think that since this really was an internal hack we can remove it.

Reply:
It is described in the API reference, can you just mark it here as deprecated (and now no longer used)?